apps/kafka/fly.toml

```diff
@@ -11,9 +11,11 @@ processes = []
   KAFKA_INTER_BROKER_LISTENER_NAME="INSIDE"
   KAFKA_HEAP_OPTS="-Xmx1024M -Xms1024M"
   KAFKA_CREATE_TOPICS="posts:10:1:compact"
+  KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR="1"
+  KAFKA_DELETE_TOPIC_ENABLE="true"
 
 [build]
-  image = "wurstmeister/kafka:latest"
+  image = "confluentinc/cp-kafka:7.3.2"
```

Comment:
For zookeeper unfortunately it was the other way around, needed to increase the memory to 1GB 😢 . Can play around to see for kafka, but i doubt that it would be less...